consumer.go

```diff
@@ -617,6 +626,12 @@ func (child *partitionConsumer) parseResponse(response *FetchResponse) ([]*Consu
 	child.fetchSize = child.conf.Consumer.Fetch.Default
 	atomic.StoreInt64(&child.highWaterMarkOffset, block.HighWaterMarkOffset)
 
+	if response.Version == 11 && len(child.consumer.conf.RackID) > 0 {
+		// we got a valid response with messages. update child's preferredReadReplica from the FetchResponseBlock
+		child.replicaInited = true
```

Comment:
this name is not descriptive enough; suggest `readReplicaConfigured` or similar.